.github/workflows/ci.yaml

```
@@ -5,63 +5,88 @@ on:
-        working-directory: sinatra  # Run tests from the correct directory
+        working-directory: sinatra
         env:
-          RACK_ENV: test  # Ensure the app runs in test mode
+          RACK_ENV: test
         run: bundle exec ruby -Itest test/app_test.rb
 
+  # Job 2: Lint Ruby Code
+  lint-ruby:
+    name: RuboCop Linting
+    runs-on: ubuntu-latest
+    needs: [] # Runs in parallel with 'test'
```

actionlint (1.7.4)

"needs" section should not be empty

(syntax-check)